Chapter2_ProgramStructure/Chapter 2 #2.js

```diff
@@ -0,0 +1,23 @@
+function f3()
+{
+    var i;
+    for(i=1;i<=100;i++)
+    {
+        if(i%3==0)
+        {
+            if(i%5==0)
+            {
+                console.log("FizzBuzz");
+            }
+            else
+            {
+                 console.log("Fizz");
+            }
+        }
+        else if(i%5==0)
+        {
+            console.log("Buzz");
+        }
+    }
+}
+f3();
```

Comment:
Redo this exercise.

Expected output:
1
2
Fizz
4
Buzz
Fizz
7
8
Fizz
Buzz
11
Fizz
13
14
FizzBuzz
(... and so on ...)


Your output:
Fizz
Buzz
Fizz
Fizz
Buzz
Fizz
FizzBuzz
Fizz
Buzz
Fizz
Fizz
Buzz
Fizz
FizzBuzz
Fizz
Buzz
Fizz
Fizz
Buzz
Fizz
FizzBuzz
Fizz
Buzz
Fizz
Fizz
Buzz
Fizz
FizzBuzz
Fizz
Buzz
Fizz
Fizz
Buzz
Fizz
FizzBuzz
Fizz
Buzz
Fizz
Fizz
Buzz
Fizz
FizzBuzz
Fizz
Buzz
Fizz
Fizz
Buzz